test/per_file_coverage.sh

```diff
@@ -49,6 +49,8 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/extensions/filters/network/sni_dynamic_forward_proxy:90.9"
 "source/extensions/health_checkers:95.9"
 "source/extensions/health_checkers/redis:95.9"
+"source/extensions/io_socket:96.0" # Death tests don't report LCOV
+"source/extensions/io_socket/user_space:96.0" # Death tests don't report LCOV
```

Comment:
What death tests?  I see no death tests in file_event_impl_test.cc

Please add some tests that call registerEventIfEmulatedEdge and unregisterEventIfEmulatedEdge to get to 100% coverage.

https://storage.googleapis.com/envoy-pr/e4c7e87/coverage/source/extensions/io_socket/user_space/file_event_impl.h.gcov.html

Reply:
registerEventIfEmulatedEdge() and unregisterEventIfEmulatedEdge() are supposed to be death test. Only IoSocketHandleImpl calls them and UserSpace::IoHandle won't call in #13418

Reply:
I add death tests and NOT_IMPLEMENTED_GCOVR_EXCL_LINE to clarify it.

Reply:
Coverage 

65/67 ~= 97.0 %

The missing two items are the 
registerEventIfEmulatedEdge and unregisterEventIfEmulatedEdge

Death tests do not help